app/jobs/zendesk_tickets_job.rb

```diff
@@ -1,11 +1,13 @@
 class ZendeskTicketsJob < ActiveJob::Base
   queue_as :zendesk
 
-  # Custom ticket fields configured in the MOJ Digital Zendesk account
+  # Custom ticket field IDs as configured in the MOJ Digital Zendesk account
   URL_FIELD = '23730083'
   SERVICE_FIELD = '23757677'
   BROWSER_FIELD = '23791776'
   PRISON_FIELD = '23984153'
+  PRISONER_NUM_FIELD = '114094604912'
+  PRISONER_DOB_FIELD = '114094604972'
```

Comment:
can you add ```.freeze``` on all these constants? (ruby allows to modify constants :trollface: )